testdata/project-v2/Makefile

```diff
@@ -54,7 +54,8 @@ generate: controller-gen
 docker-build: test
 	docker build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+	cd config/manager && kustomize edit set image controller=${IMG} && cd ../..
```

Comment:
Don't need the final `&& cd ../..` - each line in a Makefile is run in its own shell so you don't need to change back to previous directory.

Reply:
changed this as well